ui/src/pages/version/components/forms/components/SelectTransformerPanel.js

```diff
@@ -25,6 +25,12 @@ export const SelectTransformerPanel = ({
       onChange("enabled")(false);
       onChange("transformer_type")(undefined);
       onChange("type_on_ui")("");
+      onChange("resource_request")({
+        min_replica: process.env.REACT_APP_ENVIRONMENT === "production" ? 2 : 0,
+        max_replica: process.env.REACT_APP_ENVIRONMENT === "production" ? 4 : 2,
+        cpu_request: "500m",
+        memory_request: "512Mi"
+      })
```

Comment:
These lines set default resource request values, which gets set [here](https://github.com/caraml-dev/merlin/blob/main/ui/src/services/transformer/Transformer.js#L21) when a version endpoint object gets [created](https://github.com/caraml-dev/merlin/blob/main/ui/src/services/version_endpoint/VersionEndpoint.js#L38), which itself gets created when a new form is created when trying to deploy a model version.

Reply:
what if its not disabled? does the `resource_request` has to be set in the `else` block as well?

Reply:
If it's not set to `None` (i.e. if a user subsequently chooses the standard, custom or feast transformer type), we can still keep the same `resource_request` values, since these values are common across all these transformer types. I feel like there probably isn't too much value in resetting the values if this is the case.

https://github.com/caraml-dev/merlin/assets/36802364/07b2aac9-5413-41fa-aac2-2548c3d10aa0